cfgov/unprocessed/apps/regulations3k/js/search.js

```diff
@@ -124,9 +120,9 @@ function handleFilter(event) {
   try {
     searchRequest.abort();
   } catch (err) {}
-  const searchContainer = queryOne('#regs3k-results');
+  const searchContainer = document.querySelector('#regs3k-results');
   const filters = document.querySelectorAll('input:checked');
-  const searchField = queryOne('input[name=q]');
+  const searchField = document.querySelector('input[name=q]');
```

Comment:
Not for this PR, but it looks like the redundant queries in this file could be reconciled. Also, the attribute name `q` looks pretty brittle.